homeassistant/components/zha/core/channels/base.py

```diff
@@ -200,6 +200,10 @@ async def async_configure(self):
 
     async def async_initialize(self, from_cache):
         """Initialize channel."""
+        if self._ch_pool.skip_configuration:
+            self._status = ChannelStatus.INITIALIZED
+            return
+
```

Comment:
I'm wondering if this is going to affect restoration of Xiaomi based sensors values after a restart???

Reply:
You are right. Maybe I should do 
`if not from_cache and self._ch_pool.skip_configuration:`

What do you think?

Reply:
That should work.